### PR TITLE
refactor(load-data): move LoadDataParameters to odc-server

### PR DESCRIPTION
**Pull request by LuckyPickleZZ**
#### What type of PR is this?
type-refactor

#### What this PR does / why we need it:
Load data depends on schedule service, so we move the LoadDataParameters to odc-server

### CODE REVIEW

File: server/integration-test/src/test/java/com/oceanbase/odc/metadb/schedule/ScheduleTaskRepositoryTest.java [hunk 1]
```diff
@@ -107,6 +107,14 @@ public void getLatestScheduleTaskByJobNameAndJobGroup() {
         Assert.equals(latestTask.getId(), latestTaskOptional.get().getId());
     }
 
+    @Test
+    public void testFindByJobNames() {
```
Comment (yizhouxw) [inline server/integration-test/src/test/java/com/oceanbase/odc/metadb/schedule/ScheduleTaskRepositoryTest.java:111]: testFindByJobNames --> findByJobNames

Comment (LuckyPickleZZ) [reply server/integration-test/src/test/java/com/oceanbase/odc/metadb/schedule/ScheduleTaskRepositoryTest.java]: done~

**Review by yizhouxw — DISMISSED** [review]
lgtm

**Review by yhilmare — DISMISSED** [review]
lgtm

**Review by yhilmare — APPROVED** [review]
lgtm

**Pull request merged**